src/modules/signer/privateKeySigner.ts

```diff
@@ -0,0 +1,8 @@
+import { Wallet } from "ethers";
+import { Signer } from "./signer.types";
+
+export class PrivateKeySigner extends Wallet implements Signer {
+    init(): Promise<void> {
+        return new Promise((resolve) => resolve());
```

Comment:
Are we still working on implementation? 
```suggestion
        return Promise.resolve();
```